cleanlab_studio/studio/inference.py

```diff
@@ -48,7 +48,7 @@ def predict(
         Args:
             batch: batch of examples to predict classes for
             return_pred_proba: whether to return predicted class probabilities for each example
-            timeout: optional parameter to set timeout for predictions in seconds (defaults to 600s)
+            timeout: optional parameter to set timeout for predictions in seconds (defaults to 930s, which is 15 mins for lambda time limit + 30s)
```

Comment:
not sure if we to explicitly mention that we're using lambdas here since this appears in the public docs

Reply:
Yes, no need to explain implementation details in docs (users shouldn't care), would be good to explain the weird number in a code comment though (so it doesn't appear on help.cleanlab.ai).